src/main/java/org/apache/commons/imaging/formats/tiff/TiffImageData.java

```diff
@@ -53,14 +53,17 @@ public boolean stripsNotTiles() {
 
         @Override
         public ImageDataReader getDataReader(final TiffDirectory directory,
-                final PhotometricInterpreter photometricInterpreter,
-                final int bitsPerPixel, final int[] bitsPerSample, final int predictor,
-                final int samplesPerPixel, final int width, final int height, final int compression,
-            final ByteOrder byteOrder) throws IOException, ImageReadException {
+          final PhotometricInterpreter photometricInterpreter,
+          final int bitsPerPixel, final int[] bitsPerSample, final int predictor,
+          final int samplesPerPixel, final int width, final int height,
+          final int compression,
+          final int planarConfiguration,
```

Comment:
@gwlucastrig last change before merging this PR. We have some enum's like `DisposalMethod`, `IptcTypes`, png `FilterType`, `TiffDirectoryType`…

I think we should create some enum like `TiffPlanarConfiguration` for this setting. Do you think that would i) make sense and ii) work?

Nothing to add besides that, we have tests, code is in order, bug appears to be fixed.

Reply:
Making planar configuration an enum would be a good idea.  I see that you self-assigned this.  Does that mean you'll be making the change?

In the future, making predictor and compression also be enum's would also make sense.  When I am reading the data-reader code I often lose track of what the numeric values indicate.

Reply:
>Making planar configuration an enum would be a good idea. I see that you self-assigned this. Does that mean you'll be making the change?

No, sorry it's just an old habit. I believe I cannot assign the issue to you as you are an external contributor. I normally assign issues & PR's to myself so other common developers are aware I'm interested in working/finishing work on some issue.